src/rf/apps/workers/process.py

```diff
@@ -146,6 +165,7 @@ def validate_image(self, record):
                         'timeout': time.time() + TIMEOUT_SECONDS,
                         'layer_id': layer_id
                     }
+                    log.info('Queue timeout job')
```

Comment:
I feel like referring to this as `timeout` is misleading. Seems like it is more of a heartbeat.